src/script/ui/onHitTopOrBottom.ts

```diff
@@ -17,12 +17,13 @@
  *
  */
 
-const onHitTopOrBottom = (element: HTMLElement | null, onHitTop: () => void, onHitBottom: () => void) => {
-  if (!element) {
+const onHitTopOrBottom = (scrollingElement: HTMLElement | null, onHitTop: () => void, onHitBottom: () => void) => {
+  if (!scrollingElement) {
     return;
   }
 
-  const onScroll = () => {
+  const onScroll = (event: Event) => {
+    const element = event.target as HTMLElement;
```

Comment:
Can you explain what is the difference between this event target and the initial `scrollingElement`? I don't think I understand why it fixes it?